examples/customer.rs

```diff
@@ -24,7 +24,7 @@ fn main() {
             id: 1,
             first_name: "John".to_string(),
             last_name: "Doe".to_string(),
-            email: "johndoe@example.com".to_string(),
+            email: Some("johndoe@example.com".to_string()),
```

Comment:
Without my change in `src/private.rs` this will print the following:
```
Customer {
    id: 1,
    first_name: "****",
    last_name: "***",
    email: Some(
        "joh****@*******.com", <--- partial redaction even though I did not specify that 
    ),
    age: **,
}
```